DNN Platform/Library/Entities/Urls/RewriterUtils.cs

```diff
@@ -35,7 +35,7 @@ namespace DotNetNuke.Entities.Urls
 {
     public class RewriterUtils
     {
-        internal static readonly Regex ServicesFrameworkRegex = new Regex("DesktopModules/.+/API/", RegexOptions.Compiled);
+        internal static readonly Regex ServicesFrameworkRegex = new Regex("desktopModules/.+/api/|api/.+/action/", RegexOptions.Compiled);
```

Comment:
This is a breaking change as the other products/module are using it. It is better to add RegexOptions.IgnoreCase option than using ToLowerInvariant() all over the place.